pxr/imaging/rprUsd/contextHelpers.cpp

```diff
@@ -62,6 +62,83 @@ const char* k_RadeonProRenderLibName = "libRadeonProRender64.dylib";
 const char* k_RadeonProRenderLibName = "libRadeonProRender64.so";
 #endif
 
+#ifdef HDRPR_ENABLE_VULKAN_INTEROP_SUPPORT
```

Comment:
Could you put this check out of HDRPR_ENABLE_VULKAN_INTEROP_SUPPORT
Basically let's always link vulkan and use it, probably we don't need HDRPR_ENABLE_VULKAN_INTEROP_SUPPORT option anymore